ftw/statusmap/tests/test_view_functional.py

```diff
@@ -119,15 +120,15 @@ def test_transistions_with_translated_start_and_end_state(self, browser):
     def test_statusmap_on_objects_having_inactive_content(self, browser):
         # Create a container which will hold the content used for this test.
         # The container does not play an important role in the test.
-        folder = create(Builder('folder').titled('Container'))
+        folder = create(Builder('folder').titled(u'Container'))
 
         # Create some content used in this test.
         create(Builder('folder')
-               .titled('Active Folder')
+               .titled(u'Active Folder')
                .within(folder))
         create(Builder('folder')
-               .titled('Inactive Folder')
-               .having(effectiveDate=DateTime() + 10)
+               .titled(u'Inactive Folder')
+               .having(effective=datetime.today() + timedelta(10))
```

Comment:
This will work unless the test is really slow. I'm a fan of https://github.com/4teamwork/ftw.testing#freezing-datetime-now since I got to know it. But we can also leave it like that...

Reply:
Since the tests will probably never last longer than 10 days, I will leave this as it is.